sdks/go/pkg/beam/combine.go

```diff
@@ -23,14 +23,14 @@ import (
 
 // Combine inserts a global Combine transform into the pipeline. It
 // expects a PCollection<T> as input where T is a concrete type.
-func Combine(s Scope, combinefn interface{}, col PCollection, opts ...Option) PCollection {
-	return Must(TryCombine(s, combinefn, col, opts...))
+func Combine(s Scope, combinefn interface{}, col PCollection) PCollection {
```

Comment:
<!--thread_id:cc_166094412_t; commit:977a4f133a10d957b90566426581193d93041321; resolved:1-->
<!--section:context-quote-->
> **wcn3** wrote:
> We're extremely confident that Combine will never need to take options? It seems we're closing a big door here.

<!--section:body-->
It's on the table to be removed. I asked around and nobody had a use case for this feature. We can always add it back -- I don't think we're closing much of a door here, rather removing super-complex dead-ish code.

Reply:
This is one of the few places we can change a Go function signature without hardship, so I'm inclined to agree.

Reply:
I would assume the combinefn itself can be parametrized (e.g. Top(N)), in which case this should be fine.